package core.exceptions;

public class IdAlreadyInUseException extends RuntimeException {
}
